polkadot/node/core/approval-voting/src/lib.rs

```diff
@@ -431,6 +447,17 @@ impl metrics::Metrics for Metrics {
 				)?,
 				registry,
 			)?,
+			assignments_gathering_time_by_stage: prometheus::register(
+				prometheus::HistogramVec::new(
+					prometheus::HistogramOpts::new(
+						"polkadot_parachain_assignments_gather_time_by_stage",
+						"The time it takes for each stage to gather enough assignments needed for approval",
```

Comment:
is the time in millis? maybe add that to the metric name:
https://prometheus.io/docs/practices/naming/

Reply:
Done